kinetic/distribution.yaml

```diff
@@ -6474,6 +6474,16 @@ repositories:
       url: https://github.com/larics/lpms_imu.git
       version: master
     status: maintained
+  lslidar_01b:
+    doc:
+      type: git
+      url: https://github.com/leishen-lidar/LS01B.git
```

Comment:
I'm a bit confused in both of these repositories.  In both, the package.xml says that it is a BSD license, but the LICENSE file at the top is GPL.  Which one are you releasing it under?

Reply:
It's my first time tried to post on the wiki, I'm sorry for the trouble that my mistake has caused you. I have synchronized the license of the root directory with the package of the package.xml in the subdirectory

Reply:
That's no problem, it can be confusing the first time you do this.  This looks good to me now, so I'll merge this.  Thanks for making the changes.